habitat/config/config.toml

```diff
@@ -242,6 +242,8 @@ server = {{ toToml cfg.email.server }}
 port = {{ toToml cfg.email.port }}
 username = {{ toToml cfg.email.username }}
 password = {{ toToml cfg.email.password }}
+custom_email_subject = {{ toToml cfg.email.custom_email_subject }}
```

Comment:
I forget what we found out with Slack config stuff, but don't we need to wrap this in a conditional otherwise a blank value will cause reticulum not to start up?